app/Console/Commands/TrainingReminderEmails.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Carbon\Carbon;
+use DB;
+use Illuminate\Console\Command;
+use Mail;
+
+class TrainingReminderEmails extends Command {
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'Training:SendReminderEmails';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Sends training session reminder emails.';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct() {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle() {
+        try {
+            $ea_appointments = [];
+            $ea_appointments = DB::connection('ea_mysql')
+                ->table('ea_appointments')
+                ->join('ea_services', 'ea_appointments.id_services', '=', 'ea_services.id')
+                ->join('ea_users AS customer_user', 'ea_appointments.id_users_customer', '=', 'customer_user.id')
+                ->join('ea_users AS staff_user', 'ea_appointments.id_users_provider', '=', 'staff_user.id')
+                ->select('customer_user.first_name AS first_name', 'customer_user.last_name AS last_name', 'customer_user.email AS email', 'ea_appointments.start_datetime AS start_datetime', 'ea_appointments.id AS appointment_id', 'ea_appointments.notes AS appointment_notes', 'ea_services.name AS service_description', 'staff_user.first_name AS staff_first_name', 'staff_user.last_name AS staff_last_name')
+                ->where('ea_appointments.start_datetime', '<=', Carbon::now('America/New_York')->addHours(24)->format('Y-m-d H:i:s'))
+                ->where('ea_appointments.start_datetime', '>', Carbon::now('America/New_York')->format('Y-m-d H:i:s'))
```

Comment:
This ensures that appointments with start times in the past are not sent a reminder...